app.js

```diff
@@ -1,15 +1,23 @@
 const express = require("express");
-const app = require(express());
-const { getTopics } = require("./controllers/controllers.news");
-
+const app = (express());
+const { getTopics,getArticleById } = require("./controllers/controllers.news");
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticleById);
+
 
+app.use("*", (req, res) => {
+    res.status(404).send({message:"not found"})
+})
 
 
 
 
 
+app.use((err, req, res) => {
+    console.log("im in 500");
+    res.status(500).send({ message: "server error" });
```

Comment:
you want to change this console.log to log the error itself so you can see what is happening and why you are ending up here